.github/workflows/integration-test.yml

```diff
@@ -572,94 +572,64 @@ jobs:
             fi
             terraform destroy --auto-approve
 
-  EC2LinuxIntegrationTest:
-    needs: [MakeBinary, StartLocalStack, GenerateTestMatrix]
-    name: 'Test'
+  OutputEnvVariables:
+    name: 'OutputEnvVariables'
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        arrays: ${{ fromJson(needs.GenerateTestMatrix.outputs.ec2_linux_matrix) }}
-    permissions:
-      id-token: write
-      contents: read
+    outputs:
+      CWA_GITHUB_TEST_REPO_NAME: ${{ steps.set-outputs.outputs.CWA_GITHUB_TEST_REPO_NAME }}
+      CWA_GITHUB_TEST_REPO_URL: ${{ steps.set-outputs.outputs.CWA_GITHUB_TEST_REPO_URL }}
+      CWA_GITHUB_TEST_REPO_BRANCH: ${{ steps.set-outputs.outputs.CWA_GITHUB_TEST_REPO_BRANCH }}
     steps:
       - uses: actions/checkout@v3
         with:
           repository: ${{env.CWA_GITHUB_TEST_REPO_NAME}}
           ref: ${{env.CWA_GITHUB_TEST_REPO_BRANCH}}
 
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          role-to-assume: ${{ env.TERRAFORM_AWS_ASSUME_ROLE }}
-          aws-region: us-west-2
-          role-duration-seconds: ${{ env.TERRAFORM_AWS_ASSUME_ROLE_DURATION }}
-
-      - name: Cache if success
-        id: ec2-linux-integration-test
-        uses: actions/cache@v3
+      - name: Set up Go 1.x
```

Comment:
nit: Don't think we need to set up Go.

Reply:
why not?

Reply:
Where is go used in this GHA? I see it just outputting some vars.

Reply:
We're not using Go in this job.

Reply:
oh for the output var one! Didn't realize that was the job you are talking about. yeah gotcha

Reply:
Will come back to this in a follow up PR